cilium/lb/main.go

```diff
@@ -123,7 +136,7 @@ func init() {
 						Usage: "Delete all entries",
 					},
 				},
-				ArgsUsage: "--all | <address> <port> <slave-index>",
+				ArgsUsage: "--all | <address>:<port>",
```

Comment:
Let's merge it like this for now. The IP:PORT notation is used for `--frontend` and `--backend` as well. Splitting it will complicate it for those cases.